distributed/shuffle/tests/test_no_pandas_import.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import annotations
```

Comment:
Thanks @gjoseph92. Can we move this into `distributed/shuffle/tests/test_shuffle_extension.py`? It seems like it doesn't merit it's own test module

Reply:
I separated it because the subprocess pattern felt a bit distinct from those unit tests, but sure.